script/hassfest/coverage.py

```diff
@@ -64,18 +74,17 @@ def validate(integrations: dict[str, Integration], config: Config) -> None:
                 not_found.append(line)
                 continue
 
-            if (
-                not line.startswith("homeassistant/components/")
-                or len(path.parts) != 4
-                or path.parts[-1] != "*"
-            ):
+            if not line.startswith("homeassistant/components/") or len(path.parts) != 4:
```

Comment:
The previous versions resulted in all lines ending with a filename being simply ignored.